app/assets/stylesheets/common.scss

```diff
@@ -890,9 +890,9 @@ div.secondary-actions {
 /* Rules for the "Welcome" page */
 .site-welcome, .site-fixthemap {
   .sprite {
-    background-image: image-url("welcome-sprite.png");
+    background-image: image-url("welcome-sprite.svg");
     background-size: 500px 250px;
-    display: block;
+    display: inline-block;
```

Comment:
this prevents `.mx-auto` from centering the sprites

![image](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/53fdd9d8-3694-4e7f-a574-1e192447ce5e)

Reply:
480cda7c3